### DISCUSSION TITLE
npm Problem

### DISCUSSION THREAD

**Discussion by vicshi06** [smartcontractkit/full-blockchain-solidity-course-py#799] [Q&A] [answered]
Hi,

I was changing my Powershell setting since it gives me this `Running Scripts is Disabled on this System in Windows 10` error message. However, when I try to run it again, I have an issue running my npm and node. It was fine before I changed my Powershell settings. What should I do? 

![image](https://user-images.githubusercontent.com/90171690/149649316-80535a4d-8041-4587-ba33-50ad750dbf0f.png)
![image](https://user-images.githubusercontent.com/90171690/149649335-556e3e5c-e7e2-4bb1-bac4-b3652267a7f7.png)

**Comment by vicshi06** [comment, marked as answer]
nvm, all resolved when I reboot VS code. coding so fun

**Reply by cromewar** [reply]
Nice catch, yeah actually this is pretty common when you change settings on your terminal you would like to restart the terminal inside your code editor as well.
